api_docs/apm_oss.mdx

```diff
@@ -0,0 +1,28 @@
+---
+id: kibApmOssPluginApi
+slug: /kibana-dev-docs/apmOssPluginApi
+title: apmOss
+image: https://source.unsplash.com/400x175/?github
+summary: API docs for the apmOss plugin
+date: 2020-11-16
+tags: ['contributor', 'dev', 'apidocs', 'kibana', 'apmOss']
+---
+
+import apmOssObj from './apm_oss.json';
+
+## Client
+
+### Setup
+<DocDefinitionList data={[apmOssObj.client.setup]}/>
+### Start
```

Comment:
I think we need an empty line here for the `###` to render as a header:

![image](https://user-images.githubusercontent.com/1329312/109550725-e4594d80-7a8c-11eb-8deb-1d399bed9143.png)

Reply:
It might be rendering incorrectly because of the react component. As we chatted offline, these docs shouldn't be viewed in github. It would be nice if I could add something that would say "These docs should be viewed in the [docs system](link)" that would only render if viewing in github.

Reply:
I get that it wouldn't be complete, but it at least wouldn't look totally broken if there were extra newlines in there:

![image](https://user-images.githubusercontent.com/1329312/109552501-15d31880-7a8f-11eb-9999-c89ea311f5ee.png)